examples/dane/src/mastra/tools/image.ts

```diff
@@ -0,0 +1,40 @@
+import { z } from 'zod';
```

Comment:
Last thing, i think this tool can be inside the stability integration too. Then you can pass the tool into the agent

Reply:
@abhiaiyer91 Im a bit worried about some of the side effects of the current imageTool
Currently it writes to disk and uses chalk logs which is pretty specific to the dane agent

Reply:
Im also looking into ways to abstract the imageTool so that we can use Dalle or Midjourney as well as Stability